frontend/packages/ceph-storage-plugin/src/constants/ocs-install.ts

```diff
@@ -5,6 +5,8 @@ export const ZONE_LABELS = [
   'failure-domain.beta.kubernetes.io/zone', // deprecated
 ];
 
+export const RACK_LABEL = ['topology.rook.io/rack'];
```

Comment:
There is no other kind of rack label.
```suggestion
export const RACK_LABEL = 'topology.rook.io/rack';
```